yt_dlp/extractor/nrk.py

```diff
@@ -14,13 +14,12 @@
     parse_iso8601,
     str_or_none,
     traverse_obj,
-    try_get,
     url_or_none,
     urljoin,
 )
 
 
-class NRKBaseIE(InfoExtractor):
+class NRKIE(InfoExtractor):
```

Comment:
Don't remove the baseclass. Don't subclass from concrete IEs, only abstract. Subclassing from concrete can cause subtle bugs and lead to problems

Reply:
Okay, I restored the base class 👍